packages/parser/lib/interfaces/IASTNode.ts

```diff
@@ -0,0 +1,6 @@
+import EvalExprTypeEnum from "../../../core/lib/enums/EvalExprTypeEnum";
+
+export default interface IASTNode {
+  type: EvalExprTypeEnum,
+  source: any
```

Comment:
`any` is fine, but it'd be really nice to have ohm's `source`

see
1. https://github.com/harc/ohm/blob/099bd7861d7493d5660dd2b034020643077399cb/ohm-js/index.d.ts#L262-L265
2. https://github.com/harc/ohm/blob/099bd7861d7493d5660dd2b034020643077399cb/ohm-js/index.d.ts#L304-L351